### ISSUE TITLE
"No images returned from this Aura Frame" Error

### ISSUE THREAD

**Issue by Duckmanjbr** [meub/aura-frame-downloader#9]
Has anyone seen this error?  It appears login is working correctly but the image array is broken.  

```
Duckmans-Mac-Mini:aura-frame-downloader-main duckman$ python download-aura-photos.py 
Login Success
Download Error: No images returned from this Aura Frame. API responded with:
{u'message': u'Not found', u'error': True}
Downloaded {total} photos
Duckmans-Mac-Mini:aura-frame-downloader-main duckman$
```

**Comment by fmgamache** [comment]
Did you ever get a resolution for this?  I'm seeing the same error.  I get a successful login, but the it looks like the frame assets array is not found in the json data returned.

**Comment by dkwalker37** [comment]
no
________________________________

**Comment by meub** [comment]
Are you sure you are using the right frame ID? It has a format that looks like this: 476300c8-98fd-429a-9d33-72a39ca9f1c2. Is that what your frame ID looks like? I just tested and it still works fine for me.

**Comment by meub** [comment]
I see the same error you describe when I enter in an invalid frame ID. Make sure you follow the instructions from the readme to get the correct frame ID. You can get the frame ID by doing the following:

 * Go to https://app.auraframes.com and log in
 * Click on the Frame name
 * Click on "View Photos" underneath the frame
 * Then grab the ID from the URL: `https://app.auraframes.com/frame/<FRAME ID HERE>`

**Comment by dkwalker37** [comment]
No, not for sure i am putting in right place, i know it says email etc but do i put it there or at another place, can you possibly screen shot yours and also i dont know exactly how to run python and make it work. i might have to get with you next week when it is cold and do some comparing. running out of time today to mess with it, got to get on road soon.
________________________________

**Comment by meub** [comment]
The ID for emaling images to your frame is totally different and will not work. You just need to use the ID that you get from your browser's URL bar after you go to https://app.auraframes.com, log in, click on the frame name and click on "View Photos" underneath the frame.  It will have have dashes in it and be the same format as this example: 476300c8-98fd-429a-9d33-72a39ca9f1c2.

**Comment by dkwalker37** [comment]
Thank you will get with you next week! Finally got some one that would respond!

________________________________
